docker-compose.yaml

```diff
@@ -97,8 +97,14 @@ services:
         mc mb minio/cdm-lake && echo 'Bucket cdm-lake created'
       else
         echo 'bucket cdm-lake already exists'
-      fi
+      fi &&
+      mc admin user add minio minio-readonly minio123 &&
+      mc admin policy create minio cdm-lake-read-only-policy /config/cdm-lake-read-only-policy.json &&
+      mc admin policy attach minio cdm-lake-read-only-policy --user=minio-readonly &&
+      echo 'CDM Read-only user and policy set'
       "
+    volumes:
+      - ./config/cdm-lake-read-only-policy.json:/config/cdm-lake-read-only-policy.json
```

Comment:
I am planning to just copy and past this policy with our minIO UI instead of mounting the file in Rancher.

Reply:
And run above commends to create the read-only user. Unless we want to build our minIO image with custom start up commands.